src/applications/vaos/new-appointment/components/PreferredDatePage.jsx

```diff
@@ -27,9 +26,11 @@ const initialSchema = {
 
 const uiSchema = {
   preferredDate: {
-    'ui:title': 'What is the earliest date you’d like to be seen?',
+    'ui:title':
+      "Tell us the earliest day you’re available and we'll try to find the date closest to your request.",
```

Comment:
Kind of a minor thing but we're using two different apostrophes here. The little curly style in `you’re` and the straight style in `we'll`. I believe most of our copy uses the curly style. 

I only noticed this because previously I wrote a bunch of unit tests with the straight apostrophe and they failed looking for text with the curly style.